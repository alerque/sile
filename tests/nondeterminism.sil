\begin[papersize=a6]{document}
\nofolios
\font[size=11pt,family=Gentium Plus]
\noindent{}Now that we understand some of what SILE is about and what it seeks to do, 
letâ€™s dive into SILE itself.
\end{document}
