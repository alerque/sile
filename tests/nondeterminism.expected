Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	29.3016
Set font 	Gentium Plus;11;400;;normal;;LTR
T	49 82 90	(Now)
Mx 	39.2781
T	87 75 68 87	(that)
Mx 	61.9287
T	90 72	(we)
Mx 	78.7355
T	88 81 71 72 85 86 87 68 81 71	(understand)
Mx 	134.6224
T	86 82 80 72	(some)
Mx 	162.2789
T	82 73	(of)
Mx 	175.1917
T	90 75 68 87	(what)
Mx 	201.8222
T	54 44 47 40	(SILE)
Mx 	224.9884
T	76 86	(is)
Mx 	236.1771
T	68 69 82 88 87	(about)
Mx 	265.9175
T	68 81 71	(and)
Mx 	14.8819
My 	42.5016
T	90 75 68 87	(what)
Mx 	40.4679
T	76 87	(it)
Mx 	50.1502
T	86 72 72 78 86	(seeks)
Mx 	77.0898
T	87 82	(to)
Mx 	89.3233
T	71 82	(do)
Mx 	100.5757
T	15	(,)
Mx 	106.0094
T	79 72 87 183 86	(letâ€™s)
Mx 	127.5403
T	71 76 89 72	(dive)
Mx 	149.6620
T	76 81 87 82	(into)
Mx 	170.9458
T	54 44 47 40	(SILE)
Mx 	193.0675
T	76 87 86 72 79 73	(itself)
Mx 	215.5670
T	17	(.)
End page
Finish
